docs/guides/construct-circuits.ipynb

```diff
@@ -349,6 +403,93 @@
     "qc_a.decompose().draw(\"mpl\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "87f1e053",
+   "metadata": {},
+   "source": [
+    "Qubits can be measured using three methods: [`measure`](/api/qiskit/qiskit.circuit.QuantumCircuit#qiskit.circuit.QuantumCircuit.measure), [`measure_all`](/api/qiskit/qiskit.circuit.QuantumCircuit#measure_all) and [`measure_active`](/api/qiskit/qiskit.circuit.QuantumCircuit#measure_active). To learn how to visualize measured results, see the [Visualize results](/run/visualize-results) page.\n",
```

Comment:
This has changed since the PR was made

```suggestion
    "Qubits can be measured using three methods: [`measure`](/api/qiskit/qiskit.circuit.QuantumCircuit#qiskit.circuit.QuantumCircuit.measure), [`measure_all`](/api/qiskit/qiskit.circuit.QuantumCircuit#measure_all) and [`measure_active`](/api/qiskit/qiskit.circuit.QuantumCircuit#measure_active). To learn how to visualize measured results, see the [Visualize results](/guides/visualize-results) page.\n",
```